score/mw/com/impl/configuration/config_parser.cpp

```diff
@@ -430,15 +510,28 @@ auto ParseLolaEventInstanceDeployment(const score::json::Any& json, LolaServiceI
 auto ParseLolaFieldInstanceDeployment(const score::json::Any& json, LolaServiceInstanceDeployment& service) noexcept
     -> void
 {
-    const auto& fields = json.As<score::json::Object>().value().get().find(kFieldsKey.data());
-    if (fields == json.As<score::json::Object>().value().get().cend())
+    auto json_obj = json.As<score::json::Object>();
+    if (!json_obj.has_value()) {
+        return;
+    }
+    const auto& json_map = json_obj.value().get();
+    const auto& fields = json_map.find(kFieldsKey.data());
+    if (fields == json_map.cend())
     {
         return;
     }
 
-    for (const auto& field : fields->second.As<score::json::List>().value().get())
+    auto fields_list = fields->second.As<score::json::List>();
+    if (!fields_list.has_value()) {
+        return;
```

Comment:
Behavioral change. Should terminate.

```suggestion
        score::mw::log::LogFatal("lola") << "Attribute" << kFieldsKey << "not parsable. Required argument.";
        SCORE_LANGUAGE_FUTURECPP_ASSERT_PRD(false);
```